### ISSUE TITLE
Cannot route firefox audio anymore

### ISSUE THREAD

**Issue by Keridos** [a-sync/audio-router#2]
With newer Firefox I cannot properly route its audio anymore. Probably because of MultiProcess stuff. When trying to route the audio playing firefox.exe I get an access denied (running as admin) 0x000000005.

**Comment by a-sync** [comment]
Could you check with 0.10.2 if the same problem stands?  
 https://github.com/audiorouterdev/audio-router/releases/download/v0.10.2/AudioRouter-0.10.2.zip
Might be an error b/c of automatic routing as mentioned in the changelog. Otherwise i have no clue tbh.

**Comment by Keridos** [comment]
Yes I have upgraded to your version since it did not work on the old one. Windows audio routing is really really annoying. On my linux I can route my audio per process wherever I want it to just using pulseaudio.

**Comment by a-sync** [comment]
I feel your pain :)
Sorry but i'm not actively working on the project, only doing minor adjustments and tweaks. (mostly picked from other forks)

You should try out wolfreak99's version. He's doing a complete refactor afaik and you might get back some decent error messages atleast:
https://github.com/wolfreak99/audio-router
https://github.com/audiorouterdev/audio-router/pull/53#issuecomment-313839717 

However, if anyone has a possible solution, i'll be more than willing to give it a try.
